indico/modules/core/controllers.py

```diff
@@ -362,3 +363,22 @@ class RHRenderMarkdown(RH):
     def _process_POST(self, source):
         html = render_markdown(source, extensions=('nl2br',))
         return jsonify(html=html)
+
+
+class SessionExpirationMixin:
+    """Return the session expiration time."""
+
+    def _process(self):
+        try:
+            expiry = str(server_to_utc(session['_expires']))
+        except KeyError:
+            expiry = None
+        return jsonify(session_expiration=expiry)
+
+
+class RHSessionExpiration(SessionExpirationMixin, RH):
+    """Return the session expiration time without refreshing the session."""
+
+
+class RHSessionRefresh(SessionExpirationMixin, RH):
+    """Return the sesssion expiration time and refresh the current session."""
```

Comment:
```suggestion
class RHSessionExpiration(SessionExpirationMixin, RH):
    """Return the session expiration time without refreshing the session."""


class RHSessionRefresh(SessionExpirationMixin, RH):
    """Return the sesssion expiration time and refresh the current session.
    
    Refreshing the session is the standard behavior of almost any endpoint.
    """
```

Just to make it very clear why we basically have the same RH twice.